js/macros.js

```diff
@@ -90,6 +90,7 @@ function getMacroExpansion (blkname, x, y) {
     const DEFINEMODEOBJ = [[0, 'definemode', x, y, [null, 1, 2, 16]], [1, ['modename', {'value': 'custom'}], 0, 0, [0]], [2, 'pitchnumber', 0, 0, [0, 3, 4]], [3, ['number', {'value': 0}], 0, 0, [2]], [4, 'pitchnumber', 0, 0, [2, 5, 6]], [5, ['number', {'value': 2}], 0, 0, [4]], [6, 'pitchnumber', 0, 0, [4, 7, 8]], [7, ['number', {'value': 4}], 0, 0, [6]], [8, 'pitchnumber', 0, 0, [6, 9, 10]], [9, ['number', {'value': 5}], 0, 0, [8]], [10, 'pitchnumber', 0, 0, [8, 11, 12]], [11, ['number', {'value': 7}], 0, 0, [10]], [12, 'pitchnumber', 0, 0, [10, 13, 14]], [13, ['number', {'value': 9}], 0, 0, [12]], [14, 'pitchnumber', 0, 0, [12, 15, null]], [15, ['number', {'value': 11}], 0, 0, [14]], [16, 'hidden', 0, 0, [0, null]]];
+    const DELETEHELP = [[0,["start",{"collapsed":false,"xcor":0,"ycor":0,"heading":0,"color":20,"shade":80,"pensize":5,"grey":157.14285714285714,"name":"start"}],404,34,[null,18,null]],[1,["makeblock",{}],496,139,[5,2,3,4,7]],[2,["text",{"value":"note"}],611,139,[1]],[3,["solfege",{"value":"sol"}],525,171,[1]],[4,["number",{"value":4}],525,202,[1]],[5,["storein",{}],418,107,[18,6,1,8]],[6,["text",{"value":"box"}],496,107,[5]],[7,["number",{"value":4}],525,234,[1]],[8,["vspace",{}],418,170,[5,9]],[9,["vspace",{}],418,202,[8,10]],[10,["vspace",{}],418,234,[9,11]],[11,["vspace",{}],418,266,[10,13]],[12,["namedbox",{"value":"box"}],507,298,[13]],[13,["runblock",{}],418,298,[11,12,20]],[14,["deleteblock",{}],418,394,[16,15,null]],[15,["namedbox",{"value":"box"}],527,394,[14]],[16,["wait",{}],418,362,[20,17,14]],[17,["number",{"value":2}],492,362,[16]],[18,["print",{}],418,75,[0,19,5]],[19,["text",{"value":"Hello Note!"}],492,75,[18]],[20,["print",{}],418,330,[13,21,16]],[21,["text",{"value":"Goodbye Note!"}],492,330,[20]]];
```

Comment:
(1) The start block should simply be: [0, 'start', x, y, [null, 18, null]]
(2) Please follow the same formatting style as the other macros.

Reply:
Okay. I will correct it in the two pull request now